doc/install.rst

```diff
@@ -26,16 +26,16 @@ You can test it via `Nightly Build`_ today.
 
 .. _Enigmail: https://enigmail.net
 .. _Thunderbird: https://www.mozilla.org/en-US/thunderbird/
-.. _`Nightly Build`: install-autocrypt-enigmail.html
+.. _`Nightly Build`: install-autocrypt-enigmail.rst
```

Comment:
Offtopic, however, as we want the link to work on the rendered homepage, the existing `.html` link is correct  (the rendered homepage has HTML-files, not reStructuredText-files). I do not know a solution that works on both, the rendered homepage and on the preview on github.